src/test/test_pidfile.cc

```diff
@@ -0,0 +1,58 @@
+#include "global/pidfile.h"
+#include "test/unit.h"
+
+#include <errno.h>
+#include <sstream>
+#include <string.h>
+
+pidfile_path[] = "test.pid";
+
+TEST(Pidfile, NoPidfile) {
+  md_config_t conf;
+  conf.pid_file[0] = '\0'; 
+  int ret;
+  ret = pidfile_open(&conf);
+  ASSERT_EQ(ret, 0);
+  ret = pidfile_write();
+  ASSERT_EQ(ret, 0);
+  ret = pidfile_remove();
+  ASSERT_EQ(ret, 0);
+}
+
+<<<<<<< HEAD
```

Comment:
It will never pass the compilation... Could you please make check before committing?